From a84e0e0632dc0a74bd439843601662a2239667c9 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Fri, 27 Jul 2018 17:51:27 +0800
Subject: [PATCH 4398/5242] MLK-19067-3: ARM: imx7ulp: support playback and
 record

commit  b33bb3b9580221d712f2f5dd2ff677f7b5ba0cda from
https://source.codeaurora.org/external/imx/linux-imx.git

support playback and record for rpmsg device

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit b06f0122c6c280eff3bb83b0dfa284ce93526ef3)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index fd760a6..4af9fea 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -213,6 +213,8 @@
 		compatible = "fsl,imx-audio-rpmsg";
 		model = "rpmsg-audio";
 		cpu-dai = <&rpmsg_i2s>;
+		rpmsg-out;
+		rpmsg-in;
 	};
 };
 
-- 
1.7.9.5

